pkg/patterns/declarative/reconciler.go

```diff
@@ -134,6 +136,40 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+
+	// dynamic config
+	dynamicClientset, err := dynamic.NewForConfig(r.config)
+	if err != nil {
+		log.Error(err,"Unable to create dynamic client")
+		return reconcile.Result{}, err
+	}
+
+	newItems := []*manifest.Object{}
```

Comment:
Just FYI, you can just do `var newItems []*manifest.Object` - the nil slice is treated like an empty array by append.  (Though if you look at my older code, I did this too!)